ogr/services/forgejo/issue.py

```diff
@@ -1,10 +1,167 @@
+    project: "forgejo.ForgejoProject"
+
+    def __init__(self, raw_issue: _issue, project: "forgejo.ForgejoProject"):
         super().__init__(raw_issue, project)
+        self._raw_issue = raw_issue
+
+    @property
+    def _index(self):
+        return self._raw_issue.number
+
+    @property
+    def title(self) -> str:
+        return self._raw_issue.title
+
+    @title.setter
+    def title(self, new_title: str) -> None:
+        self._issue_api_call(
+            self.project.service.api.issue.edit_issue,
+            title=new_title,
+        )
+
+    @property
+    def id(self) -> int:
+        return self._raw_issue.id
+
+    @property
+    def url(self) -> str:
+        return self._raw_issue.url
+
+    @property
+    def description(self) -> str:
+        return self._raw_issue.body
+
+    @property
+    def author(self) -> str:
+        return self._raw_issue.user.login
+
+    @property
+    def created(self) -> datetime:
+        return self._raw_issue.created_at
+
+    @property
+    def status(self) -> IssueStatus:
+        return IssueStatus[self._raw_issue.state]
+
+    @property
+    def assignees(self) -> list:
+        try:
+            return self._raw_issue.assignees
+        except AttributeError:
+            return None
+
+    @staticmethod
+    def create(
+        project: "forgejo.ForgejoProject",
+        title: str,
+        body: str,
+        private: Optional[bool] = None,
+        labels: Optional[list[str]] = None,
+        assignees: Optional[list[str]] = None,
+    ) -> "Issue":
+
+        if private:
+            raise NotImplementedError()
+        if not project.has_issues:
+            raise IssueTrackerDisabled()
+
+        issue = project.service.api.issue.create_issue(
+            owner=project.namespace,
+            repo=project.repo,
+            title=title,
+            body=body,
+            labels=labels,
+            assignees=assignees,
+        )
+        return ForgejoIssue(issue, project)
+
+    @staticmethod
+    def get(project: "forgejo.ForgejoProject", issue_id: int) -> _issue:
+        if not project.has_issues:
+            raise IssueTrackerDisabled
+
+        try:
+            issue = project.service.api.issue.get_issue(
+                owner=project.namespace, repo=project.repo, index=issue_id,
+            )
+        except Exception as ex:
+            raise OperationNotSupported(f"Issue {issue_id} not found") from ex
+        return ForgejoIssue(issue, project)
+
+    @staticmethod
+    def get_list(
+        project: "forgejo.ForgejoProject",
+        status: IssueStatus = IssueStatus.open,
+        author: Optional[str] = None,
+        assignee: Optional[str] = None,
+        labels: Optional[list[str]] = None,
+    ) -> list["Issue"]:
+        if not project.has_issues:
+            raise IssueTrackerDisabled
+
+        parameters: dict[str, Union[str, list[str], bool]] = {
+            "state": status if status != IssueStatus.open else "open",
```

Comment:
I'm not really sure if this produces what you think it does… looking at the definition of `IssueStatus`, it is an `IntEnum`, I think you'll find out when you start testing it.